common/decisions/00_major_decisions_german_culture.txt

```diff
@@ -0,0 +1,65 @@
+form_german_culture_decision = {
+	picture = {
+		reference = "gfx/interface/illustrations/decisions/decision_dynasty_house.dds"
+	}
+	desc = form_german_culture_decision_desc
+	selection_tooltip = form_german_culture_decision_tooltip
+	decision_group_type = major
+
+	ai_check_interval = 120
+
+	is_shown = {
+		is_ruler = yes
+		is_playable_character = yes
+		highest_held_title_tier > 3
+
+		root = {
+			OR = {
+				culture = culture:bavarian
+				culture = culture:dutch
+				culture = culture:franconian
+				culture = culture:frisian
+				culture = culture:german
+				culture = culture:langobard
+				culture = culture:saxon
+				culture = culture:swabian
+			}
+		}
+	}
+
+	is_valid = {
+			completely_controls_region = german_unified_lands
+	}
+
+	is_valid_showing_failures_only = {
+		is_landed = yes
+		is_capable_adult = yes
+		is_imprisoned = no
+		is_at_war = no
+	}
+
+	effect = {
+		gain_heroic_legend_seed_tooltip_effect = yes
+		custom_tooltip = form_german_culture_decision_effect_tooltip
+	
+		convert_family_culture_and_notify_vassals_effect = {
+			CONVERTER = root
+			OLD_CULTURE = root.culture
+			NEW_CULTURE = culture:unified_german	
+		}
+
+		hidden_effect = {
+			root.culture = {
+				get_all_innovations_from = root.culture
+			}
```

Comment:
Very close, but be careful because the syntax matters:
`culture:unified_german` is needed as shown below as that's the culture you are applying this to.

```
culture:unified_german = {
	get_all_innovations_from = root.culture
}
```

Reply:
This should be done FIRST, before any culture related alterations take effect